app/models/topic_tracking_state.rb

```diff
@@ -52,7 +52,6 @@ def self.publish_new(topic)
       last_read_post_number: nil,
       highest_post_number: 1,
       created_at: topic.created_at,
-      topic_id: topic.id,
```

Comment:
This is not used on the client side and the attribute is already sent as part of the payload in messageBus one level up.